### DISCUSSION TITLE
Can DecontX be used for scATAC-seq data ?

### DISCUSSION THREAD

**Discussion by dweemx** [campbio/celda#315] [Q&A] [answered]
Hi,

Thanks for the great package. It has already been useful for decontaminating some scRNA-seq data and seem to do the job quite well.

I was wondering if DecontX could be used for scATAC-seq ? I've seen from the paper that the model does assume some assumptions (e.g.: distrbution of the data). Do you think those assumptions would still be valid for scATAC-seq data ?

**Comment by joshua-d-campbell** [comment, marked as answer]
Hi @dweemx, we haven't extensively tested it on single cell ATAC-seq data yet. However, if you have a count matrix of chromatin peaks x cells, then I think it should still work in principle. You may want to supply your own cell cluster labels via the 'z' parameter if the default method for quick cell clustering does not work well. It is on our to-do list to try it out on other single cell data types.
